### PR TITLE
Solved issue #22

### PR DESCRIPTION
**Pull request by cr2025x1**
https://github.com/intelsdi-x/snap-plugin-publisher-kafka/issues/22

### CODE REVIEW

**Comment by cr2025x1** [comment]
'make' command right after cloning SmartX-Team/snap-plugin-publisher-kafka when complete clean state(emtpy $GOPATH/src directory) does not yield the fixed version. It will use the unfixed version from the original repo, intelsdi-x/snap-plugin-publisher-kafka. This can be identified while loading the built plugin binary. If the version is 7, then this happened.

Either copying kafka/kafka.go to cloned intelsdi-x/snap-plugin-publisher-kafka,
or modifying the line 26 from cloned SmartX-Team/snap-plugin-publisher-kafka's main.go;
"github.com/intelsdi-x/snap-plugin-publisher-kafka/kafka" --> "github.com/SmartX-Team/snap-plugin-publisher-kafka/kafka"

Only then the proper build will be done.

**Comment by IzabellaRaulin** [comment]
@cr2025x1, is this pull request ready to be reviewed? I do not why the travis tests didn't trigger. Could You push this commit once again?

**Comment by cr2025x1** [comment]
Done as requested. TravisCI was notified and tests are underway.

File: main.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	"os"
 
 	// Import the snap plugin library
-	"github.com/intelsdi-x/snap-plugin-publisher-kafka/kafka"
+	"github.com/SmartX-Team/snap-plugin-publisher-kafka/kafka"
```
Comment (IzabellaRaulin) [inline main.go]: Please, revert this change - there should be "github.com/intelsdi-x/snap-plugin-publisher-kafka/kafka" as import path

**Comment by IzabellaRaulin** [comment]
Thanks a lot for triggering Travis - please, revert your change in main.go, that is why travis checks have failed.

**Comment by IzabellaRaulin** [comment]
For me it looks good. One question - do we really need 5 commits? Would you mind to squash these to one?

**Comment by cr2025x1** [comment]
I didn't know that is possible. 😄 Done as requested.

**Comment by IzabellaRaulin** [comment]
Thanks a lot. LGTM
